edi_voxel_sale_order_import/models/sale_order.py

```diff
@@ -257,8 +257,9 @@ def _parse_product_list_data_voxel(self, order, xml_root, error_msgs):
         for line_element in line_elements:
             line_vals = {"order_id": order.id}
             self._parse_product_voxel(line_vals, line_element)
+            self._parse_qty_uom_voxel(line_vals, line_element)
             line_vals = so_line_obj.play_onchanges(
-                line_vals, ["product_id", "product_uom_qty"],
+                line_vals, self._get_so_line_onchange_fields(),
```

Comment:
Instead of having to add this and inherit for each change, I think you can just check for the added keys in `line_vals`, and pass them to the method.